README.md

```diff
@@ -1 +1,135 @@
-# module-1-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [IWES conference 2023] <a name="about-project"></a>
+
+- This project is called IWES-Conference-2023. It is based on a conference that is to happen towards the end of this year in Abuja, Nigeria for women all over the world. This project contains all information about the coming conference and the previous ones as well.
+
+**[IWES conference 2023]** is a website that is used to tell women about this program and at the same time invite them for the program.
+
+## 🛠 Built With <a name="HTML and CSS and JS"></a>
+
+### Tech Stack <a name="Front end (Javascript)"></a>
+
+- This is a mobile and desk-top browser responsive website.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.w3schools.com/js/">JS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[It is a two paged website ]**
+- **[It has a navbar for navigation to the pages]**
+- **[There is a join button that when focused on tells you the amount of slots remaining for the program]**
+- **[This website is mobile and desktop responsive and this conclusion came from testing it on most devices]**
+- **[There is an active class that shows what page you are currently on]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+🚀 Live Demo
+
+- (https://otegaa.github.io/IWES-Conference-2023/)
+
```

Comment:
_**[OPTIONAL]**_ You can create an inline link by wrapping link text in brackets [ ], and then wrapping the URL in parentheses ( ) 🤗
e.g `Live Demo [GitHub Pages](https://pages.github.com/)` ==> Live Demo [GitHub Pages](https://pages.github.com/)

You can learn more on **Basic writing and formatting syntax - GitHub Docs** [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#:~:text=You%20can%20create%20an%20inline,K%20to%20create%20a%20link.)